### ISSUE TITLE
Linux: Command line doesn't function

### ISSUE THREAD

**Issue by danieljnilson** [qgeissmann/OpenCFU#22]
Running Kubuntu 16.10

I installed OpenCFU and used it's GUI a few times on some test images. Everything worked fine, played with all the settings, and successfully outputted the results to CSVs.

However, when I try to use the terminal (since I want to do batch analyses over 100> plates in the future), I get the the following error:

`daniel@Celebrate:~$ opencfu -i /home/daniel/Documents/Gdrive/LabWork/Viability/OpenCFU/20161117_160653.jpg -m auto -d bil -t 3 -r 1 -G 3
ERROR setting-up the INPUT file"/home/daniel/Documents/Gdrive/LabWork/Viability/OpenCFU/20161117_160653.jpg".`

Any idea how I may debug this?

Thanks, it's a really cool piece of software.

**Comment by qgeissmann** [comment]
Thanks for reporting that @danieljnilson! I will try to reproduce/debug that later this week. Could you upload your image, in case it is specific to it ?
Cheers,

**Comment by danieljnilson** [comment]
![20161117_160653](https://cloud.githubusercontent.com/assets/6232748/20495877/c2f7df8c-afd7-11e6-928c-12f72bba6ae6.jpg)

Here's the image. I re-created the error this morning. Again, the terminal approach works only when the image is placed in the home folder.

Cheers

**Comment by qgeissmann** [comment]
do you mean if works as expected if you run cd before? E.g.:

```
$ cd  /home/daniel/Documents/Gdrive/LabWork/Viability/OpenCFU
$ opencfu -i 20161117_160653.jpg -m auto -d bil -t 3 -r 1 -G 3
```

**Comment by danieljnilson** [comment]
No; it does not work if I cd into the image directory first. Only if the image is in the home directory does the CLI function.

However on a fresh Ubuntu 16.04 VM, the OpenCFU CLI doesn't seem to care where the image is as long I either cd into the directory or I call the file with its full address (such as in my OP). Weird. I presume this might be a compiling derived issue? I don't know enough about compiling to determine what's different between my Kubuntu 16.10 and the Ubuntu 16.04 VM. This may be a niche issue. If you want any system information from me, I'd be happy to provide it but I don't consider it a OpenCFU issue at this point and my VM is a functional&stable work around.

**Comment by DMaisterra** [comment]
Get into the github account.. cloned it.
autoreconfig -i
get all the depependencies.... libs... and dev versions. ./configure, make, install... your done!.
hint, try to get libs now, dont compile libs. bye.
